libraries/chain/include/eosio/chain/kv_chainbase_objects.hpp

```diff
@@ -30,6 +50,16 @@ namespace eosio { namespace chain {
                                                  member<kv_object, shared_blob, &kv_object::kv_key>>,
                                    composite_key_compare<std::less<name>, std::less<name>, unsigned_blob_less>>>>;
 
+   inline void use_rocksdb_for_disk(chainbase::database& db) {
```

Comment:
Since this is defined in the header file, it has to be inline to avoid separate cpp files' including it resulting in linker complaints about multiple implementations. It may not show up in the checking build, since you may get "lucky" and the multiple includes may end up in the same unity build constructed group compile it creates.